### PR TITLE
avoiding fixed persistenceId on example

### PR DESCRIPTION
**Pull request by franciscolopezsancho**
`PersistenceId` is explained right after in the docs, as @patriknw  pointed out, so this probably suffices.

### CODE REVIEW

**Review by johanandren — APPROVED** [review]

**Comment by johanandren** [comment]
OK TO TEST

File: akka-persistence-typed/src/test/scala/docs/akka/persistence/typed/BasicPersistentBehaviorCompileOnly.scala [hunk 1]
```diff
@@ -108,10 +108,10 @@ object BasicPersistentBehaviorCompileOnly {
   import MyPersistentBehavior._
 
   object RecoveryBehavior {
-    def apply(): Behavior[Command] =
+    def apply(persistenceId: PersistenceId): Behavior[Command] =
       //#recovery
       EventSourcedBehavior[Command, Event, State](
-        persistenceId = PersistenceId.ofUniqueId("abc"),
+        persistenceId = persistenceId,
```
Comment (patriknw) [inline akka-persistence-typed/src/test/scala/docs/akka/persistence/typed/BasicPersistentBehaviorCompileOnly.scala:114]: pls change corresponding in the Java code also: BasicPersistentBehaviorTest.java

**Review by patriknw — COMMENTED** [review]
Java also...

File: akka-persistence-typed/src/test/scala/docs/akka/persistence/typed/BasicPersistentBehaviorCompileOnly.scala (hunk 1 above)
Comment (franciscolopezsancho) [reply akka-persistence-typed/src/test/scala/docs/akka/persistence/typed/BasicPersistentBehaviorCompileOnly.scala:114]: Done now

**Review by johanandren — APPROVED** [review]

**Review by patriknw — APPROVED** [review]
LGTM

**Pull request merged**